downstream/modules/eda/proc-eda-view-activation-output.adoc

```diff
@@ -5,7 +5,7 @@
 You can view the output of the activations in the *History* tab.
 
 .Procedure
-. Select the *History* tab to access the list of all the activation instances.
+. Select the btn:[History] tab to access the list of all the activation instances.
```

Comment:
I don't think we use the btn:[] approach for tabs. i think the original is correct.